tests/backend/broker/backpressure-overflow/sender.zeek

```diff
@@ -0,0 +1,60 @@
+# The sender fires batches of one-way ping events to a receiver to make it
+# consume buffer space. An included epoch counter tracks how often we'receiving
+# backpressure-induced unpeerings. While this is happening, the sender also
+# receives ping events (of a different type) from the manager, which it echoes
+# back to it. This verifies that the sender itself does not lock up. Messaging
+# to stdout provides status updates.
+
+# Where to send the ping. See zeek_init() below, which adapts this
+# depending on whether we generate load worker->proxy or proxy->worker.
+global ping_topic = Cluster::proxy_topic;
```

Comment:
Does this help to have the information just in one place? Maybe also in manager.zeek.

```suggestion
global ping_topic = 	getenv("SENDER") == "proxy" ? Cluster::worker_topic : Cluster::proxy_topic;
```

Reply:
Yep, I like it! Thanks.